common/assets.go

```diff
@@ -0,0 +1,109 @@
+package common
+
+/*          All the assets on pegnet
+ *
+ *          PegNet,                 PNT,        PNT
+ *
+ *          US Dollar,              USD,        pUSD
+ *          Euro,                   EUR,        pEUR
+ *          Japanese Yen,           JPY,        pJPY
+ *          Pound Sterling,         GBP,        pGBP
+ *          Canadian Dollar,        CAD,        pCAD
+ *          Swiss Franc,            CHF,        pCHF
+ *          Indian Rupee,           INR,        pINR
+ *          Singapore Dollar,       SGD,        pSGD
+ *          Chinese Yuan,           CNY,        pCNY
+ *          Hong Kong Dollar,       HKD,        pHKD
+ * DROPPED  Tiawanese Dollar,       TWD,        pTWD
```

Comment:
dropped? For what reason? And what's the motivation to keep it in the code then?

Reply:
They are listed in the whitepaper, but the data source we poll is missing them. So I removed them, and figured there would need be a conversation as to make a new polling source or actually remove them.

Reply:
As for motivation, I think it would be good to fix and keep them vs remove them. But that is a business decision that is above me